mango-cli/src/main/webapp/js/overall.js

```diff
@@ -33,12 +33,14 @@ function render(refName, start, end) {
   // Features
   if (featuresExist) {
     renderFeatures(refName, start, end);
+  } else {
+    $("#featArea").css( ' display', 'none' );
   }
 
   // Variants
   if (variantsExist) {
     renderVariants(refName, start, end);
-  }
+  } else }
```

Comment:
This breaks the javascript. I'll fix it later, but this won't allow anything to be displayed in overall right now.